### DISCUSSION TITLE
Add inline toolbar

### DISCUSSION THREAD

**Discussion by PrimeTimeTran** [saadeghi/daisyui#2628] [Ideas / feature requests]
Would love to be able to add a toolbar to the left or right of the tabs inline. 

It feels like wasted/dead space if it's empty.

An example is most browsers. 

The inline toolbar should be able to be button group or collapse/expand icon/icons. 

<img width="1127" alt="Screenshot 2023-12-07 at 3 58 38 PM" src="https://github.com/saadeghi/daisyui/assets/16382165/cebea86b-6f48-4118-a20b-084ad2987dcd">

Button groups also work/would be nice.

https://experience.sap.com/fiori-design-web/toolbar-overview/#responsiveness
 
<img width="806" alt="Screenshot 2023-12-07 at 3 57 00 PM" src="https://github.com/saadeghi/daisyui/assets/16382165/3f926adf-8592-4214-9362-5a0056661bb0">

Thanks for your work!

**Comment by saadeghi** [comment]
I think this pattern is more a combination of 2 things. Tabs, and button/link/etc on the other side.

I can think of 2 ways to do this with daisyUI:  
https://play.tailwindcss.com/3ezuIM2IGc

Let me know if you have a question.

**Reply by PrimeTimeTran** [reply]
Appreciate your help. This breaks current behavior though.

Could I get you to add the right toolbar to a documentation example? 

It's this example from the [site](https://daisyui.com/components/tab/) so it'd be awesome if this additional feature(right toolbar) could work without breaking that & not requiring JS.

```html
<div role="tablist" class="tabs tabs-lifted">
  <input type="radio" name="my_tabs_2" role="tab" class="tab" aria-label="Tab 1" />
  <div role="tabpanel" class="tab-content bg-base-100 border-base-300 rounded-box p-6">Tab content 1</div>

  <input type="radio" name="my_tabs_2" role="tab" class="tab" aria-label="Tab 2" checked />
  <div role="tabpanel" class="tab-content bg-base-100 border-base-300 rounded-box p-6">Tab content 2</div>

  <input type="radio" name="my_tabs_2" role="tab" class="tab" aria-label="Tab 3" />
  <div role="tabpanel" class="tab-content bg-base-100 border-base-300 rounded-box p-6">Tab content 3</div>
</div>
```

I've been able to swap out the tabs using JS but that makes the code less awesome. 

To keep it 100% one of the main reasons I switched from Flowbite is because I was blown away by how you were able to achieve so much without JS.

Thanks in advance. Appreciate your work.

**Reply by saadeghi** [reply]
Thanks!

How about something like this?

https://play.tailwindcss.com/gEwar9UoWi

**Reply by PrimeTimeTran** [reply]
Thanks for your effort, I super appreciate it.
